docs/contributor/development.md

```diff
@@ -0,0 +1,76 @@
+# Development
+
+Telemetry Manager has been bootstrapped with [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) 3.6.0. 
+Too add more APIs, use [Kubebuilder](https://book.kubebuilder.io/cronjob-tutorial/new-api.html).
+
+## Prerequisites
+
+- Install [kubebuilder 3.6.0](https://github.com/kubernetes-sigs/kubebuilder), which is the base framework for the Telemetry manager. Required to add new APIs.
+- Install [Golang 1.19](https://golang.org/dl/) or newer (for development and local execution).
```

Comment:
Since it's a new docu, let's write Golang 1.20

Reply:
fixed